**Linux with security manager** [Build 3685](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=80750&buildTypeId=WF_PullRequest_LinuxSm) outcome was **FAILURE** using a merge of 1cd65a10bb9901642a2756790bffd44a002ffd96
Summary: Tests failed: 3 (3 new), passed: 935, ignored: 291 Build time: 00:07:56
### Failed tests
```
org.wildfly.mod_cluster.undertow.UndertowContextTestCase.getPath: java.lang.AssertionError: expected same:</> was not:<>
	at org.wildfly.mod_cluster.undertow.UndertowContextTestCase.getPath(UndertowContextTestCase.java:68)


org.wildfly.mod_cluster.undertow.UndertowHostTestCase.getContexts: java.lang.AssertionError: expected same:</> was not:<>
	at org.wildfly.mod_cluster.undertow.UndertowHostTestCase.getContexts(UndertowHostTestCase.java:84)


org.wildfly.mod_cluster.undertow.UndertowHostTestCase.findContext: java.lang.AssertionError: expected same:</> was not:<>
	at org.wildfly.mod_cluster.undertow.UndertowHostTestCase.findContext(UndertowHostTestCase.java:103)


```

--- a/mod_cluster/undertow/src/main/java/org/wildfly/mod_cluster/undertow/UndertowContext.java
+++ b/mod_cluster/undertow/src/main/java/org/wildfly/mod_cluster/undertow/UndertowContext.java
@@ -59,7 +59,8 @@ public class UndertowContext implements Context {
 
     @Override
     public String getPath() {
-        return this.deployment.getDeploymentInfo().getContextPath();
+        String path = this.deployment.getDeploymentInfo().getContextPath();
+        return "/".equals(path) ? "" : path;
     }
 
     @Override

--- a/mod_cluster/undertow/src/main/java/org/wildfly/mod_cluster/undertow/UndertowHost.java
+++ b/mod_cluster/undertow/src/main/java/org/wildfly/mod_cluster/undertow/UndertowHost.java
@@ -93,8 +93,10 @@ public class UndertowHost implements Host {
 
     @Override
     public Context findContext(String path) {
+        String findPath = "".equals(path) ? "/" : path;
+
         for (Deployment deployment : this.host.getDeployments()) {
-            if (deployment.getDeploymentInfo().getContextPath().equals(path)) {
+            if (deployment.getDeploymentInfo().getContextPath().equals(findPath)) {
                 return new UndertowContext(deployment, this);
             }
         }

--- a/mod_cluster/undertow/src/test/java/org/wildfly/mod_cluster/undertow/UndertowContextTestCase.java
+++ b/mod_cluster/undertow/src/test/java/org/wildfly/mod_cluster/undertow/UndertowContextTestCase.java
@@ -58,7 +58,7 @@ public class UndertowContextTestCase {
     @Test
     public void getPath() {
         DeploymentInfo info = new DeploymentInfo();
-        String expected = "/";
+        String expected = "";
         info.setContextPath(expected);
         
         when(this.deployment.getDeploymentInfo()).thenReturn(info);

--- a/mod_cluster/undertow/src/test/java/org/wildfly/mod_cluster/undertow/UndertowHostTestCase.java
+++ b/mod_cluster/undertow/src/test/java/org/wildfly/mod_cluster/undertow/UndertowHostTestCase.java
@@ -68,7 +68,7 @@ public class UndertowHostTestCase {
     public void getContexts() {
         Deployment deployment = mock(Deployment.class);
         DeploymentInfo info = new DeploymentInfo();
-        String expectedPath = "/";
+        String expectedPath = "";
         info.setContextPath(expectedPath);
         HttpHandler handler = mock(HttpHandler.class);
         
@@ -89,7 +89,7 @@ public class UndertowHostTestCase {
     public void findContext() {
         Deployment deployment = mock(Deployment.class);
         DeploymentInfo info = new DeploymentInfo();
-        String expectedPath = "/";
+        String expectedPath = "";
         info.setContextPath(expectedPath);
         HttpHandler handler = mock(HttpHandler.class);
         

